gears/src/baseapp/errors.rs

```diff
@@ -41,6 +41,39 @@ impl RunTxError {
 
 #[derive(Debug, thiserror::Error)]
 pub enum QueryError {
+    #[error("query path not found")]
+    PathNotFound,
+    #[error("Block height must be greater than or equal to zero")]
+    InvalidRequest,
```

Comment:
Is there some other reason why the request can be invalid?

Reply:
Changed to `InvalidHeight`